src/components/PlusButtonEdge.tsx

```diff
@@ -5,13 +5,15 @@ import { useIntegrationJsonStore, useNestedStepsStore, useVisualizationStore } f
 import { IStepProps, IVizStepNode } from '@kaoto/types';
 import { Popover, Tooltip } from '@patternfly/react-core';
 import { PlusIcon } from '@patternfly/react-icons';
-import { getBezierPath, Position, useReactFlow } from 'reactflow';
+import { ReactNode } from 'react';
+import { EdgeText, getBezierPath, Position, useReactFlow } from 'reactflow';
 
 const foreignObjectSize = 40;
 
 export interface IPlusButtonEdge {
   data?: any;
   id: string;
+  label?: string | ReactNode;
```

Comment:
maybe the same here :)

```suggestion
  label?: ReactNode;
```